### ISSUE TITLE
Autocomplete behave weird when gateway is selected

### ISSUE THREAD

**Issue by Pooya-Oladazimi** [ts4nfdi/terminology-service-suite#142]
How to reproduce:
1- select the API gateway as the endpoint. 
2- type anything like "milk"
3- then remove it with backspace and type it again

It shows an incorrect set of results and also the term label loses its boldness. 

It seems it happens only when the gateway is selected.

can you reproduce this? 

cc: @jusa3 @VincentKneip

**Comment by jusa3** [comment]
Yes. It seems to depend on how fast you're typing. If you type "milk" very fast, it just loads the results for "m" or "mil".
